src/src/PluginStructs/P128_data_struct.cpp

```diff
@@ -963,7 +965,7 @@ void P128_data_struct::rainbow(void) {
   float progress = (float)zaehler / (float)fadetime;
```

Comment:
Why is there a variable name in German?

Reply:
Inherited code... migrated from the ESPEasyPlayGround